### PR TITLE
update all examples to use proper data urls

### LINKED ISSUE #289: Data URLs showed in the examples are incorrect
The data URLs showed this spec are all using a semicolon `;` to separate the media type from the data. According to [RFC2397 Section 3](https://www.rfc-editor.org/rfc/rfc2397.html#section-3), it should be a comma `,` instead.

### PR DESCRIPTION
**Pull request by decentralgabe**
fix #289 

cc: @timothee-haudebourg

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/290.html" title="Last updated on Aug 13, 2024, 1:43 AM UTC (0af32f3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/290/8230909...0af32f3.html" title="Last updated on Aug 13, 2024, 1:43 AM UTC (0af32f3)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2156,12 +2156,12 @@ <h2 id="presentations">Presentations</h2>
     },
     {
       "@context": "https://www.w3.org/ns/credentials/v2",
-      "id": "data:application/vc-ld+jwt;eyVjV...RMjU",
+      "id": "data:application/vc-ld+jwt;base64url,ebyVjV...RMjU",
```
Comment (brentzundel) [inline index.html:2159]: did a 'b' get added here accidentally or on purpose?

Comment (decentralgabe) [reply index.html:2159]: nice spot, this was an accident

Comment (decentralgabe) [inline index.html:2159]:
```suggestion
      "id": "data:application/vc-ld+jwt;base64url,eyVjV...RMjU",
```

Comment (decentralgabe) [reply index.html]: fixed https://github.com/w3c/vc-jose-cose/pull/290/commits/4dbee0948a14a90e852cd7c7a47164eef095a7eb

**Review by brentzundel — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Comment by decentralgabe** [comment]
This is an example change (class 1) - and has approval from editors, so I will be merging it now.

File: index.html (hunk 1 above)
Comment (dlongley) [reply index.html]: I don't think this is right, the `base64url` should not be present -- the media type here is already for a text-based format.

Comment (decentralgabe) [reply index.html]: @dlongley in what was merged I removed `base64url` and just kept `,`

**Pull request merged**